testing/cli/test_cli_sct_analyze_lesion.py

```diff
@@ -1,16 +1,71 @@
-import os
-
 import pytest
 import logging
 
-from spinalcordtoolbox.scripts import sct_analyze_lesion
+import pickle
+import numpy as np
+
+from spinalcordtoolbox.utils import sct_test_path, extract_fname
+from spinalcordtoolbox.scripts import sct_analyze_lesion, sct_label_utils
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def dummy_lesion(tmp_path):
+    """Define a fake voxel lesion using the specified dimensions."""
+    # TODO: Parametrize this fixture to test other lesion dimensions
+    starting_coord = [29, 27, 25]
+    dim = [1, 4, 1]
```

Comment:
Also, I tried to write the fixture in such a way that it should be easy to play around with the dimensions to test lesions of differing sizes. :)